core/src/main/java/org/stellar/anchor/sep6/ExchangeAmountsCalculator.java

```diff
@@ -124,4 +132,42 @@ public static class Amounts {
     String amountFee;
     String amountFeeAsset;
   }
+
+  private String getClientId(Sep10Jwt token) throws BadRequestException {
```

Comment:
💡 This function should not belong to `ExchangeAmountsCalculator` class. Can we move it somewhere else?

Reply:
Yeah, I'll move it into a separate class since it will be used by SEP-24 as well.